.terraform.lock.hcl

```diff
@@ -1,43 +0,0 @@
-# This file is maintained automatically by "terraform init".
```

Comment:
Why remove this?

Reply:
I misunderstood the file and thought it was unnecessary, I've re-added it back in